### PR TITLE
adding notifications panel

### PR DESCRIPTION
**Pull request by filipeotero**
* replacing @hig/notifications-flyout to @filipeop/notifications-panel
* defining development mode into module.exports (warning message was appearing)

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -2,6 +2,7 @@ const path = require('path');
 const TerserPlugin = require('terser-webpack-plugin');
 
 module.exports = {
+    mode: 'development',
```
Comment (QilongTang) [inline webpack.config.js:5]: I had an PR locally to set it to `none` and forgot to open it. Please use `none` as we are forcing modes in script section in packages.json.

File: package.json [hunk 2]
```diff
@@ -28,10 +28,10 @@
     "react-dom": "^18.2.0"
   },
   "devDependencies": {
+    "@filipeop/notifications-panel": "^0.0.2",
```
Comment (QilongTang) [inline package.json:31]: Please increase the "version": "0.0.7", to 0.0.8 so we can release and publish for Dynamo

**Review by QilongTang — COMMENTED** [review]
A few comments then LGTM

File: webpack.config.js [hunk 3]
```diff
@@ -2,6 +2,7 @@ const path = require('path');
 const TerserPlugin = require('terser-webpack-plugin');
 
 module.exports = {
+    mode: 'node',
```
Comment (QilongTang) [inline webpack.config.js:5]: https://webpack.js.org/configuration/mode/

Should be `none`

**Comment by QilongTang** [comment]
One more comment then LGTM

File: webpack.config.js (hunk 3 above)
Comment (filipeotero) [reply webpack.config.js]: Thanks for noticing! Updated.

**Review by QilongTang — APPROVED** [review]

**Pull request merged**